server/controllers/medical/patients/index.js

```diff
@@ -437,6 +437,12 @@ function find(options) {
   filters.dateTo('dateBirthTo', 'dob');
   filters.equals('health_zone');
   filters.equals('health_area');
+
+  // filters for location
+  const orignSql = `(originVillage.name LIKE ?) OR (originSector.name LIKE ?) OR (originProvince.name LIKE ?)`;
+  let village = options.originVillageName || ``;
```

Comment:
Just FYI, you could do this in one line with a combination of [lodash's `_.fill()` method](https://lodash.com/docs/4.17.5#fill) and [JavaScript Template Strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) like so:
```js
const params = _.fill(3, `%${options.originVillageName || ''}%`);
filters.customMultiParameters('originVillageName', originSQL, params);
```